app/views/categories/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<section class='mainCover'>
+  <section class='mainCoverTwo'>
+    <section class='header'>
+    <div class='headerOne'>
+      <div class="imageElement"> <%= image_tag("menu.png", :alt => "menu feed", class: "imageElement")%> </div>
+      <div>Transactions</div>
+      <div><%= image_tag("search.png", :alt => "menu feed", class: "imageElement")%></div>
+    </div>
+    <div class="headerTwo">
+      <div>Most Recent</div>
+      <div>Most ancient</div>
+    </div>
+  </section>
+      <div class="mainContainer">
+  <section class="cartListCover">
+          <% if @categories.empty?%>
+            <p class=" ">You don't have any categories yet !!</p>
+          <% else %>
+            <ul class="cartListContainer">
+              <% @categories.each do |cat| %>
+                  <%= link_to category_spendings_path(category_id: cat.id), class:'link' do %>
+                      <li class="cartList">
+                          <div class=""><%= cat.icon %></div>
+                          <div class=" "><%= cat.name %></div>
+                          <div class=" ">$ <%= cat.spendings.sum(:amount)%>
+                      </li>
+                  <% end %>
+              <% end %>
+            </ul>
+          <% end %>
+  </section>
+  <div class='categoryButton userSubmitContainer'><%= link_to 'Create New Category', new_category_path, class: '' %>
+  </div>
+</div>
```

Comment:
- [ ] It would be nice if you could match your design with the provided design and place the button at the bottom of the screen just as it is in the original design. Kindly work on that 👍🏽
- [ ] Also, I encourage you to add the required background color for the button to match the given design. 👍🏽

**The Original Design**
<img width="335" alt="Screenshot 2023-10-21 at 5 02 07 PM" src="https://github.com/tonnytech/Budget_App/assets/50941074/19c16349-ee91-4ea1-b291-d7a04dc1b7d6">

**Your Design**
<img width="434" alt="Screenshot 2023-10-21 at 5 16 21 PM" src="https://github.com/tonnytech/Budget_App/assets/50941074/b1b3fac5-57af-4aec-a9a3-6d206add783d">